src/main/java/com/hackathonorganizer/hackathonwriteservice/team/controller/TagController.java

```diff
@@ -0,0 +1,38 @@
+package com.hackathonorganizer.hackathonwriteservice.team.controller;
+
+import com.hackathonorganizer.hackathonwriteservice.team.model.Tag;
+import com.hackathonorganizer.hackathonwriteservice.team.model.dto.TagRequest;
+import com.hackathonorganizer.hackathonwriteservice.team.service.TagService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/tags")
+@Slf4j
+public class TagController {
```

Comment:
Proponuje dopisac testy

Reply:
Testy będą tylko nie mam czasu już dziś na to XD